server/chat-plugins/scavengers.ts

```diff
@@ -747,14 +891,14 @@ class ScavengerHunt extends Rooms.RoomGame {
 
 	onUpdateConnection() {}
 
-	onChatMessage(msg) {
-		let msgId = toID(msg);
+	onChatMessage(msg: string) {
+		let msgId = toID(msg) as string;
 
 		const commandMatch = msg.match(SCAVENGE_REGEX);
 		if (commandMatch) msgId = msgId.slice(toID(commandMatch[0]).length);
 
 		const filtered = this.questions.some(q => {
-			return q.answer.some(a => {
+			return q.answer.some((a: string) => {
```

Comment:
Apparently this is because `this.questions` is an array of `AnyObject`. Can you add the type definition now?

Reply:
(It's not a big deal if you don't want to.)